### PR TITLE
Version substitute in jaseci kit

### PR DESCRIPTION
**Pull request by ashagraw91**
## Describe your changes

Make use of version number for github for jaseci KIT and substitute specif version number

## Link to related issue
https://app.clickup.com/t/2dtqwpf

## Does this introduce a new feature or change the existing feature of Jaseci? If so, please add tests.
yes

## Will this impact Jaseci users? If so, please write a paragraph describing the impact.
Should have no impact

## Anything in this PR should the Jaseci developer/contributor community pay particular attention to? If so, please describe.

The versioning of the jaseci KIT while deployments

### CODE REVIEW

**Review by asimsalim — CHANGES_REQUESTED** [review]
@ashagraw91  Can we create new yaml files as discussed ?
It can be in the format use_enc.prod.yaml 
This is also required for Jaseci.yaml

**Comment by ypkang** [comment]
@asimsalim @ashagraw91 I like this change. Just curious -- what is the original problem/use case that lead to this change? 
And, yea I agree it would be good to keep the option of installing from source.

File: jaseci_kit/jaseci_kit/modules/use_enc/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Install Specific version of jaseci KIT model in the POD
+
+To install Specific version of jaseci KIT, Follow below process.
+The use_enc.yaml ConfigMAP has environmnet variable declared as $version , which we will substitute while applying the Yaml 
+
+
+### Export environment variable 
+
+```bash
+export version= {version no}  i.e 1.3.3.16
```
Comment (marsninja) [inline jaseci_kit/jaseci_kit/modules/use_enc/README.md]: @ashagraw91 I'd use a bit more specific env var like "jskit_version" throughout

**Comment by ashagraw91** [comment]
@ypkang this change is because the new implementation requires specific version of jaseci kit to be deployed in a pod, keeping it latest breaks changes if there has been a major update in kit model. the version helps in keeping in the stable model for ai pods

**Comment by ashagraw91** [comment]
@asimsalim I have updated for use_qa and use_enc. jaseci willnot need to change as the pod Ip for both will same for jaseci pod. Also let me know if other yamls also need to be updated?

**Review by ypkang — APPROVED** [review]
This looks good to me.

**Review by marsninja — APPROVED** [review]
LGTM

**Review by marsninja — APPROVED** [review]
LGTM

**Pull request merged**